pkg/orchestrator/util.go

```diff
@@ -37,20 +39,13 @@ const (
 	K8sStatefulSet
 )
 
-var (
-	telemetryTags = map[NodeType][]string{
-		K8sCluster:     getTelemetryTags(K8sCluster),
-		K8sCronJob:     getTelemetryTags(K8sCronJob),
-		K8sDeployment:  getTelemetryTags(K8sDeployment),
-		K8sJob:         getTelemetryTags(K8sJob),
-		K8sNode:        getTelemetryTags(K8sNode),
-		K8sPod:         getTelemetryTags(K8sPod),
-		K8sReplicaSet:  getTelemetryTags(K8sReplicaSet),
-		K8sService:     getTelemetryTags(K8sService),
-		K8sDaemonSet:   getTelemetryTags(K8sDaemonSet),
-		K8sStatefulSet: getTelemetryTags(K8sStatefulSet),
+var telemetryTags = map[NodeType][]string{}
+
+func init() {
+	for _, nodeType := range NodeTypes() {
+		telemetryTags[nodeType] = getTelemetryTags(nodeType)
```

Comment:
Could we get rid of this map and have the function using it calling `getTelemetryTags(nodeType)` directly ?